### PR TITLE
GH74: Build on Mono

### PR DESCRIPTION
**Pull request by mholo65**

### CODE REVIEW

**Comment by mholo65** [comment]
@devlead how to hook up our bitrise builds to this repo?

**Comment by gep13** [comment]
@mholo65 If you know what sort of project you want created, I can get them added to Bitrise, and then I can add you as an admin of the project.

**Comment by mholo65** [comment]
@gep13 Thanks! Similar as what we currently doing with Cake should be sufficient.

File: setup.cake [hunk 1]
```diff
@@ -155,7 +156,7 @@ BuildParameters.Tasks.DotNetCoreBuildTask
 
             // If .NET Core project, copy using dotnet publish for each target framework
             // Otherwise just copy
-            if(parsedProject.IsNetCore)
+            if(parsedProject.IsVS2017ProjectFormat)
```
Comment (gep13) [inline setup.cake]: @mholo65 why did you make this change?  What was the requirement?  Could this be the same issue that @devlead is facing when using Cake.Recipe in a few projects that he has?

Comment (mholo65) [reply setup.cake]: I noticed that after `Cake.Recipe`was updated (and indirectly `Cake.Incubator`) `dotnet publish` wasn't run anymore for `Cake.Bakery` due to the fact that it's not a NETCore project. I.e. I only target `net461`... Did a corresponding PR to Cake.Recipe, here https://github.com/cake-contrib/Cake.Recipe/pull/205

Comment (mholo65) [reply setup.cake]: But due to the face that we don't have a solution for https://github.com/cake-contrib/Cake.Recipe/pull/176 yet, I need to use my own copy of `DotNetCore-Build` in order to build under Mono,

**Pull request merged**